docsource/buildinitclient.rst

```diff
@@ -76,50 +76,64 @@ Build the client::
 
     BUILD SUCCESSFUL
     Total time: 3 seconds
-    
+
 The client jar is created in ``dist/client/WorkspaceClient.jar``.
 
 For simplicity, copy the required jars into a single directory::
 
     bareubuntu@bu:~/ws$ mkdir tryjavaclient
     bareubuntu@bu:~/ws$ cd tryjavaclient/
     bareubuntu@bu:~/ws/tryjavaclient$ cp ../workspace_deluxe/dist/client/WorkspaceClient.jar .
-    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/jackson/jackson-*-2.2.3.jar .
-    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/kbase/auth/kbase-auth-0.3.1.jar .
-    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/kbase/common/kbase-common-0.0.10.jar .
+    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/jackson/jackson-*-2.5.4.jar .
```

Comment:
The workspace proper is still using jackson 2.2.3 and from what I recall, there was a problem with updating. If the clients work with 2.5.4 I guess this is fine, but something doesn't seem quite right about having different jackson versions for the server and client... but I can't articulate it so maybe this is ok.